data/settings/src/main/java/com/google/jetpackcamera/settings/model/TargetFrameRate.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.jetpackcamera.settings.model
+
+import android.util.Range
+import com.google.jetpackcamera.settings.TargetFrameRate as TargetFrameRateProto
+
+enum class TargetFrameRate(val range: Range<Int>) {
+    TARGET_FPS_NONE(Range(0, 0)),
+    TARGET_FPS_15(Range(15, 15)),
+    TARGET_FPS_30(Range(30, 30)),
+    TARGET_FPS_60(Range(60, 60)),
+    TARGET_FPS_100(Range(100, 100)); // excessively high option to target the supported upper ranges
```

Comment:
There won't be any devices that support this. I don't think we should even call it out as an option.

Reply:
Gotcha. I think it might cause more confusion than anything. Even if it's choosing the highest possible FPS, it's still not choosing "100fps", which is what a user who is not familiar with cameras or CameraX would expect. Thanks for taking it out.